samples/ballerina-by-example/examples/transformers/transformers.bal

```diff
@@ -7,46 +7,47 @@ struct Employee {
 
 @Description {value:"Defining Person struct."}
 struct Person {
-    string first_name;
-    string last_name;
+    string firstName;
+    string lastName;
     int age;
     string city;
+    string street;
 }
 
 @Description{value:"Defining a default transformer for converting from 'Person' type to 'Employee' type."}
 transformer <Person p, Employee e> {
-  e.name = p.first_name.toLowerCase();
-  e.age = p.age;
-  e.address = p.city.toLowerCase();
+    e.name = p.firstName + " " + p.lastName;
+    e.age = p.age;
+    e.address = p.street + "," + p.city.toUpperCase();
 }
 
 @Description{value:"Defining a named transformer for converting from 'Person' type to 'Employee' type."}
-transformer <Person p, Employee e> Foo() {
-  e.name = p.first_name.toLowerCase();
-  e.age = p.age + 2;
-  e.address = p.city.toUpperCase();
+transformer <Person p, Employee e> incrementAge() {
```

Comment:
Remove this one, it's too artificial to say, you're incrementing the age when transforming from person to employee.